package.json

```diff
@@ -28,6 +28,17 @@
     "engines": {
         "node": ">=4.2.0"
     },
+    "files": [
+        "./bin",
+        "./lib",
+        "!./lib/enu",
```

Comment:
Do we have any idea what `enu` is?

Reply:
I believe these are the XML files that the localization team consumes. Our localize task produces both the output we use as well as these files to give to them to use in some GUI program.

Reply:
Now, I'm not actually sure that this is copied in the `lkg` build; I will check.

(Personally, I think that these localization team things would be better suited for a different task, as we don't need it in our dev workflow day-to-day, but that's a topic for another PR.)